src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IReleasePublishingStatusService.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces
 {
-    public interface IReleaseStatusService
+    public interface IReleasePublishingStatusService
```

Comment:
Does this service want to be renamed? I thought it was just the ReleaseStatusService in the Publisher that should be renamed?

Reply:
Yes, it's because it still refers to the Publisher's Release Statuses (if you have a gander at the internals of the class, you'll see all the Table Storage stuff) :) .